src/EditorFeatures/CSharpTest/CodeActions/ConvertConversionOperators/ConvertConversionOperatorsTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ConvertConversionOperators;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings.ConvertConversionOperators
+{
+    [Trait(Traits.Feature, Traits.Features.ConvertConversionOperators)]
+    public class ConvertConversionOperatorsTests : AbstractCSharpCodeActionTest
+    {
+        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
+            => new CSharpConvertConversionOperatorsRefactoringProvider();
```

Comment:
all new features/tests should use hte new VerifyCS harness (see elsewhere in this project)